frontend/src/pages/associations/sidebar.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { api } from "../../services/apiService";
+import { SideBar, SideBarItem } from "../../utils/sidebar";
+import { useQuery } from "react-query";
+
+const ListPagesItem = ({ pages, association }) =>
+    pages.map(page => (
+        <SideBarItem
+            icon={"book"}
+            to={`/associations/${association.id}/pages/${page.id}`}
+            key={page.id}
+        >
+            {page.title}
+        </SideBarItem>
+    ));
+
+const AddPageItem = ({ association }) => {
+    if (!association.myRole.pagePermission) {
+        return null;
+    }
+
+    return (
+        <SideBarItem
+            icon={"plus"}
+            to={`/associations/${association.id}/pages/new`}
+        >
+            Ajouter une page
+        </SideBarItem>
+    );
+};
+
+export const AssociationSidebar = ({ association }) => {
```

Comment:
`Sidebar` or `SideBar`?
Actually, _sidebar_ is [a word](https://www.wordreference.com/enfr/sidebar), please change